build/docker/dos-service-fuzzy-search-api/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM 461183108257.dkr.ecr.eu-west-2.amazonaws.com/uec/service-finder/api-base:1.0.20191119
+
+ADD assets/application/dos-service-fuzzy-search-api.jar /application/
```

Comment:
Shall we use `COPY` instead of `ADD`? Recommendation is to use `ADD` only when we have to.

Reply:
Done